src/pages/EntityDetails/_entity-details.scss

```diff
@@ -212,79 +212,84 @@
   }
 
   .entity-details__apps {
-    th,
-    td {
-      &:nth-child(1) {
-        width: 20%;
-      }
-
-      &:nth-child(2) {
-        width: 15%;
-      }
-
-      &:nth-child(3) {
-        width: 15%;
-      }
-
-      &:nth-child(4) {
-        width: 10%;
-      }
+    &:not(.selectable) {
+      th,
+      td {
+        // Version.
+        &:nth-child(3) {
+          width: 10%;
+        }
 
-      &:nth-child(5) {
-        width: 15%;
-      }
+        // Scale.
+        &:nth-child(4) {
+          width: 10%;
+        }
 
-      &:nth-child(6) {
-        width: 10%;
-      }
+        @include mobile-and-medium {
+          // Scale.
+          &:nth-child(4) {
+            width: 15%;
+          }
 
-      &:nth-child(7) {
-        width: 10%;
-      }
+          // Version.
+          &:nth-child(3),
+          // Revision.
+          &:nth-child(5),
+          // Store.
+          &:nth-child(6) {
+            display: none;
+          }
+        }
 
-      &:nth-child(8) {
-        width: 25%;
+        @include mobile {
+          // Message.
+          &:nth-child(7) {
+            display: none;
+          }
+        }
       }
     }
   }
 
   .entity-details__apps.selectable {
```

Comment:
Just a small thought, but was wondering if it might make sense to move this after line 251 as `&.selectable`?

Reply:
Done.